.travis/setup.sh

```diff
@@ -29,3 +29,13 @@ if [[ $LANG == *"php"* ]]; then
   sudo apt-get update
   sudo apt-get install --no-install-recommends -y php
 fi
+
+# Install Dart. (https://dart.dev/get-dart)
+if [[ $LANG == *"dart"* ]]; then
+  sudo apt-get update
+  sudo apt-get install apt-transport-https
```

Comment:
I believe Travis has already installed that package and we can save some CI time by removing these lines.
```
The following packages will be upgraded:
  apt-transport-https
1 upgraded, 0 newly installed, 0 to remove and 253 not upgraded.
Need to get 26.5 kB of archives.
After this operation, 0 B of additional disk space will be used.
Get:1 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 apt-transport-https amd64 1.2.32 [26.5 kB]
Fetched 26.5 kB in 0s (89.4 kB/s)
(Reading database ... 126205 files and directories currently installed.)
Preparing to unpack .../apt-transport-https_1.2.32_amd64.deb ...
Unpacking apt-transport-https (1.2.32) over (1.2.31) ...
Setting up apt-transport-https (1.2.32) ...
```